documentation/book/proc-bridge-seeking-offsets-for-partition.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-bridge-seeking-offset-for-partition-{context}']
+= Seeking to offsets for a partition
+
+Next, use the link:https://strimzi.io/docs/bridge/latest/#_seek[positions^] endpoints to configure the Kafka Bridge consumer to retrieve messages for a partition from a specific offset, and then from the latest offset. This is referred to in Apache Kafka as a seek operation.
+
+.Procedure
+
+. Seek to a specific offset for partition 0 of the `quickstart-bridge-topic` topic:
++
+[source,curl,subs=attributes+]
+----
+curl -X POST http://localhost:8080/consumers/bridge-quickstart-consumer-group/instances/bridge-quickstart-consumer/positions \
+  -H 'content-type: application/vnd.kafka.v2+json' \
+  -d '{
+    "offsets": [
+        {
+            "topic": "bridge-quickstart-topic",
+            "partition": 0,
+            "offset": 2
+        }
+    ]
+}'
+----
++
+If the request is successful, the Kafka Bridge returns a `204` code only.
```

Comment:
could it be worth to do a poll records to show that the client re-reads records from the offset where you seek?

Reply:
Good suggestion. I've added an extra step here.